.github/workflows/macos.yml

```diff
@@ -77,39 +77,28 @@ jobs:
             wget --quiet https://gstreamer.freedesktop.org/data/pkg/osx/${{ env.GST_VERSION }}/gstreamer-1.0-devel-${{ env.GST_VERSION }}-universal.pkg
             wget --quiet https://gstreamer.freedesktop.org/data/pkg/osx/${{ env.GST_VERSION }}/gstreamer-1.0-${{ env.GST_VERSION }}-universal.pkg
             for package in *.pkg ;
-              do sudo installer -verbose -pkg "$package" -target /
+            do sudo installer -verbose -pkg "$package" -target /
             done
-            echo "PKG_CONFIG_PATH=/Library/Frameworks/GStreamer.framework/lib/pkgconfig:${{ env.PKG_CONFIG_PATH }}" >> "$GITHUB_ENV"
```

Comment:
can you restore this line, just in case?

Reply:
The same thing is already done here: https://github.com/mavlink/qgroundcontrol/blob/master/libs/qmlglsink/CMakeLists.txt#L129